Surface_sweep_2/examples/Surface_sweep_2/plane_sweep.cpp

```diff
@@ -1,18 +1,16 @@
-//! \file examples/Arrangement_on_surface_2/sweep_line.cpp
+//! \file examples/Arrangement_on_surface_2/plane_sweep.cpp
 // Computing intersection points among curves using the sweep line.
 
-#include <CGAL/Cartesian.h>
-#include <CGAL/MP_Float.h>
-#include <CGAL/Quotient.h>
-#include <CGAL/Arr_segment_traits_2.h>
-#include <CGAL/Sweep_line_2_algorithms.h>
 #include <list>
 
-typedef CGAL::Quotient<CGAL::MP_Float>                  NT;
-typedef CGAL::Cartesian<NT>                             Kernel;
-typedef Kernel::Point_2                                 Point_2;
-typedef CGAL::Arr_segment_traits_2<Kernel>              Traits_2;
-typedef Traits_2::Curve_2                               Segment_2;
+#include <CGAL/Exact_predicates_exact_constructions_kernel.h>
+#include <CGAL/Arr_segment_traits_2.h>
+#include <CGAL/Surface_sweep_2_algorithms.h>
```

Comment:
The header renaming is a breaking change that must be announced in Installation.html + the former header should still be kept with a redirect to the new header. You can use macros for this purpose such as in `AABB_tree/include/CGAL/AABB_polyhedron_triangle_primitive.h` for example.